pages/clubs-guides/clubs-guides.jsx

```diff
@@ -0,0 +1,92 @@
+var React = require('react');
+var HeroUnit = require('../../components/hero-unit.jsx');
+
+var GuideList = React.createClass({
```

Comment:
I wonder if this should be a separate component in a separate file. Either way, it's probably not important right now.